nixos/modules/installer/cd-dvd/installation-cd-base.nix

```
@@ -46,5 +46,11 @@ with lib;
     done
   '';
 
+  environment.defaultPackages = with pkgs; [
+    # Include some version control tools.
+    git
+    rsync
+  ];
```

Nit:

```suggestion
  environment.defaultPackages = with pkgs; [
    rsync
  ];

  programs.git.enable = true;
```

This way, the git package that is installed can be configured, and setting `programs.git.package` won't result in two `git`s being in `defaultPackages`.

---

fair points, but if you're doing that it should probably `mkDefault`'d

---

> fair points, but if you're doing that it should probably mkDefault'd

Updated

---

Updated it.